preprocessing/tabular_data/training_model_on_tabular_data.ipynb

```diff
@@ -0,0 +1,299 @@
+{
```

Comment:
<p> </p><p>Suggestion to rephrase second sentence:  </p><blockquote>"We preprocessed this data in the preprocessing_tabular_data.ipynb and persisted it using storemagic."</blockquote><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>